src/teamwork/__tests__/e2e.teamwork.test.ts

```diff
@@ -1,26 +1,15 @@
 import { IMerjoonProjects, IMerjoonTasks, IMerjoonUsers } from "../../common/types";
 import { TeamworkTransformer } from '../transformer';
 import { TeamworkService } from '../service';
-import { ITeamworkConfig } from '../types';
 import { TeamworkApi } from '../api';
 import { ID_REGEX } from "../../utils/regex";
+import { getConfig} from "../config";
 
 describe('e2e TeamWork', () => {
   let service: TeamworkService;
 
   beforeEach(() => {
-    const {
```

Comment:
let's move lines 12-26 to new file `teamwork-service.ts`

and `export teamworkService = new TeamworkService(api, transformer);`